BeardedSpice/TuneInStrategy.m

```diff
@@ -0,0 +1,53 @@
+//
+//  TuneInStrategy.m
+//  BeardedSpice
+//
+//  Created by Cristian Yáñez on 14-01-15.
+//  Copyright (c) 2015 Tyler Rhodes / Jose Falcon. All rights reserved.
+//
+
+#import "TuneInStrategy.h"
+
+@implementation TuneInStrategy
+-(id) init
+{
+    self = [super init];
+    if (self) {
+        predicate = [NSPredicate predicateWithFormat:@"SELF LIKE[c] '*tunein.com*'"];
+    }
+    return self;
+}
+-(BOOL) accepts:(id <Tab>)tab
+{
+    return [predicate evaluateWithObject:[tab URL]];
+}
+-(NSString *) toggle
+{
+    return @"document.querySelector('.playbutton-cont').click();";
```

Comment:
Sorry I modified the code from another Strategy and what I understood from that code is that as there seems not to be a dedicated control for pausing, instead of doing nothing when pressing pause and already paused it would toggle.

Anyway, I tried it and works fine (Not very experienced in Objective-C nor Cocoa programming, sorry)